packages/client-ui/tsconfig.json

```diff
@@ -4,6 +4,6 @@
     "outDir": "dist",
     "rootDir": "src"
   },
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "src/**/*.json"],
```

Comment:
Not needed anymore since you are including `ipynb`?

Reply:
Yeah, this shouldn't be necessary any longer.